community/modules/scripts/ramble-execute/README.md

```diff
@@ -0,0 +1,101 @@
+## Description
+
+This module will create a startup-script runner that will execute Ramble commands.
+
+Ramble is a multi-platform experimentation framework capable of driving
+software installation, acquiring input files, configuring experiments, and
+extracting results. For more information about Ramble, see:
+https://github.com/GoogleCloudPlatform/ramble
+
+This module outputs a startup script runner, which can be combined with other
+startup script runners to execute a set of Ramble commands.
+
+Ramble makes extensive use of Spack. It must be installed with a Toolkit runner
+generated by the [spack module](../spack-install/README.md) following the
+[basic example](#basic-example) below.
+
+> **_NOTE:_** This is an experimental module and the functionality and
+> documentation will likely be updated in the near future. This module has only
+> been tested in limited capacity.
+
+# Examples
+
+## Basic Example
+
+Below is a basic example of using this module.
+
+```yaml
+- id: spack
```

Comment:
nit: indent the example by 2 spaces. In a blueprint they will be indented so it makes it easier to copy paste.